test/integration/msi_int_windows_test.go

```diff
@@ -0,0 +1,127 @@
+package integration
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/ActiveState/cli/internal/environment"
+	"github.com/ActiveState/cli/internal/testhelpers/e2e"
+	"github.com/autarch/testify/assert"
+	"github.com/autarch/testify/require"
+)
+
+var (
+	msiDir = mustFilepathByProjectRoot(`/build/msi`)
+	logDir = mustFilepathByProjectRoot(`/build`)
+
+	msiExt        = ".msi"
+	perlMsiPrefix = "ActivePerl"
+
+	asToken = "ActiveState"
+
+	checkPerlVersionCmd = "perl -v"
+	checkPerlModulesCmd = "perldoc -l DBD::Pg"
```

Comment:
Please incorporate this into the tests itself so far as that is feasible. Right now I dislike the amount of scanning I have to do to grok the code. Constantly scrolling up to reference some thing that really could've just been right there in the code.